pandas/core/groupby/generic.py

```diff
@@ -377,8 +383,8 @@ def _get_index() -> Index:
             result = Series(data=values, index=_get_index(), name=self._selection_name)
             return self._reindex_output(result)
 
-    def _aggregate_named(self, func, *args, **kwargs):
-        result = OrderedDict()
+    def _aggregate_named(self, func, *args, **kwargs) -> Dict:
+        result = {}  # type: Dict
```

Comment:
can use the new style of typing now, e.g. ```result: Dict = {}```, though odd that mypy can't infer that. better to be even specific, Dict[str, <whatisthis>]

Reply:
Just FYI mypy will always complain for an empty container creation as it can't infer the subtypes